pkg/cmd/roachtest/test_runner.go

```diff
@@ -1309,7 +1310,19 @@ func (r *testRunner) runTest(
 	case <-time.After(timeout):
 		// NB: We're adding the timeout failure intentionally without cancelling the context
 		// to capture as much state as possible during artifact collection.
-		t.addFailure(0, "test timed out (%s)", timeout)
+		timeoutErr := registry.ErrorWithOwnership{
+			Err: errors.Newf("test timed out (%s)", timeout),
```

Comment:
Not sure this works? We don't take empty `Owner` fields into account when an error has ownership assigned. I think it's safer to only assign ownership (i.e., set the error to an instance of `ErrorWithOwnership`) when we have an owner to set.

Reply:
Yeah you're right, it didn't appear to error out but it wouldn't assign an owner + it was messing with the test that Stan mentioned above.